pkgs/misc/sagetex/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, pkgs, lib, ... }:
+
+stdenv.mkDerivation rec {
+  version = "3.6";
+  pname = "sagetex";
+  passthru.tlType = "run";
+
+  src = pkgs.fetchurl {
+    url = "https://github.com/sagemath/sagetex/releases/download/v${version}/sagetex-${version}.tar.gz";
+    sha256 = "fzol58VztvRLJHaaMYQ8gmMvSa5Lu2AZJ74buDPle4E=";
+  };
+
+  installPhase = ''
+    path="$out/tex/latex/sagetex"
+    mkdir -p "$path"
+    cp -va *.sty *.cfg *.def "$path/"
+  '';
+
+  meta = with lib; {
+    description = "embed code, results of computations, and plots from Sage into LaTeX documents";
+    license = licenses.gpl2Plus;
+    maintainers = with maintainers; [
+      alexnortung
+    ];
```

```suggestion
    maintainers = with maintainers; [ alexnortung ];
```
Looks a bit cleaner IMO